src/shogun/mathematics/linalg/internal/implementation/Add.h

```diff
@@ -128,6 +129,7 @@ struct add<Backend::NATIVE, Matrix>
 	static void compute(T* A, T* B, T* C,
 		T alpha, T beta, index_t len)
 	{
+#pragma omp parallel for shared(A, B, C, alpha, beta)
```

Comment:
@lisitsyn @lambday shall i remove this then? And is there still a need of benchmarking if we remove this?